tests/test_instantiator.py

```diff
@@ -60,6 +60,63 @@ def test_interpolation_weight_width_class(data_dir):
     assert font.info.openTypeOS2WidthClass == 9
 
 
+def test_default_groups_only(data_dir, caplog):
+    """Test that only the default source's groups end up in instances."""
+
+    d = designspaceLib.DesignSpaceDocument()
+    d.addAxisDescriptor(
+        name="Weight", tag="wght", minimum=300, default=300, maximum=900
+    )
+    d.addSourceDescriptor(location={"Weight": 300}, font=ufoLib2.Font())
+    d.addSourceDescriptor(location={"Weight": 900}, font=ufoLib2.Font())
+    d.addInstanceDescriptor(styleName="2", location={"Weight": 400})
+    d.findDefault()
```

Comment:
Is this necessary?

Reply:
Yes, so set the `default` property.

Reply:
Mh. It is done on load in dsLib usually, just not when we construct the objects ourselves.

Reply:
We should make that default a cached property

Reply:
It needs to be recomputed every time you change sources or axis data. Maybe making it a simple just-in-time lookup is enough?